node/node.go

```diff
@@ -368,7 +368,9 @@ func (n *Node) startHTTP(endpoint string, apis []rpc.API, modules []string, cors
 	if err != nil {
 		return err
 	}
-	n.log.Info("HTTP endpoint opened", "url", fmt.Sprintf("http://%s", endpoint), "cors", strings.Join(cors, ","), "vhosts", strings.Join(vhosts, ","))
+	n.log.Info("HTTP endpoint opened", "url", fmt.Sprintf("http://%v/", listener.Addr()),
```

Comment:
Is the trailing `/` accidental or deliberate?

Reply:
accidental... :/

Reply:
No wait, it's actually correct to use a trailing slash there. I think it's the more correct URL, since it's a `GET / HTTP/1,1` -- a domain is not a valid url